proposals/date_shfit.json

```diff
@@ -0,0 +1,86 @@
+{
+    "id": "date_shfit",
+    "summary": "Calculates and manipulates dates and times",
+    "description": "Allows to calculate a new date and time based on a given date and time by adding or subtracting a given temporal period.",
+    "categories": [
+        "date & time"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The base date and time to manipulate.\n\nThe millisecond part of the date and time is optional and defaults to 0 if not given. The time zone never changes.",
```

Comment:
maybe this is not the right place to raise this issue: but shouldn't we require dates and times to be in UTC always?

Reply:
I would require UTC at API level and leave time zone conversions at client lib level (at best)

Reply:
We can't require that right now (breaking). I'm even unsure that's a good idea. On the one hand it's much simpler, on the other hand, there could be use cases where timezones may be important. We'd need to closely check with all domains we want to support in openEO (climatology for example) whether timezones are important for whatever reason.

Reply:
Looking at the RFC again I can at least say that DST can't be an issue. From the numerical timezones, we can't figure out whether DST has an effect or not. So we ignore DST completely here. Numerical time zones are allowed though (e.g. +02:00), but those can easily be converted to UTC.